tests/integration/test_run.py

```diff
@@ -0,0 +1,11 @@
+import os
+import pytest
+from mlflow import projects
+
+pytestmark = pytest.mark.integration
+
+
+def test_simple_run():
+    projects.run(
+        os.path.join(os.path.dirname(__file__), "..", "resources", "example_yarn_project"),
+        backend="yarn", entry_point="greeter", parameters={"name": "test_name"})

```

Comment:
I would have expected some asserts here, at least to check that a yarn job is launched and the greeter is executed

Reply:
Added the asserts. The status is handled by projects.run. If it doesn't succeeded it raises an exception.